src/pages/deposit.vue

```diff
@@ -412,6 +412,7 @@ export default Vue.extend({
       if (token.symbol.toLowerCase() !== "eth") {
         const wallet = walletData.get().syncWallet;
         const tokenAddress = wallet!.provider.tokenSet.resolveTokenAddress(token.symbol);
+        // @ts-ignore
         const erc20contract = new Contract(tokenAddress, IERC20_INTERFACE, wallet!.ethSigner);
```

Comment:
Do not ignore ts errors, infer type from the Contract

Reply:
@JackHamer09 but this particular case should be fixed (**no @ts-ignore please**)